SPECS/nvml/nvml.spec

```diff
@@ -0,0 +1,725 @@
+The libpmemblk implements a pmem-resident array of blocks, all the same
+size, where a block is updated atomically with respect to power
+failure or program interruption (no torn blocks).
+
+For example, a program keeping a cache of fixed-size objects in pmem
+might find this library useful. This library is provided for cases
+requiring large arrays of objects at least 512 bytes each. Most
+developers will find higher level libraries like libpmemobj to be
+more generally useful.
+
+%package -n libpmemblk
+Summary:        Persistent Memory Resident Array of Blocks library
+Requires:       libpmem >= %{version}-%{release}
+
+%description -n libpmemblk
+The libpmemblk implements a pmem-resident array of blocks, all the same
+size, where a block is updated atomically with respect to power
+failure or program interruption (no torn blocks).
+
+%package -n libpmemblk-debug
+Summary:        Debug variant of the Persistent Memory Resident Array of Blocks library
+Requires:       libpmemblk = %{version}-%{release}
+
+%description -n libpmemblk-debug
+The libpmemblk implements a pmem-resident array of blocks, all the same
+size, where a block is updated atomically with respect to power
+failure or program interruption (no torn blocks).
+
+This sub-package contains debug variant of the library, providing
+run-time assertions and trace points. The typical way to access the
+debug version is to set the environment variable LD_LIBRARY_PATH to
+%{_lib64dir}/pmdk_debug.
+
+%package -n libpmemlog
+Summary:        Persistent Memory Resident Log File library
+Requires:       libpmem >= %{version}-%{release}
+
+%description -n libpmemlog
+The libpmemlog library provides a pmem-resident log file. This is
+useful for programs like databases that append frequently to a log
+file.
+
+%package -n libpmemlog-devel
+Summary:        Development files for the Persistent Memory Resident Log File library
+Requires:       libpmem-devel = %{version}-%{release}
+Requires:       libpmemlog = %{version}-%{release}
+
+%description -n libpmemlog-devel
+The libpmemlog library provides a pmem-resident log file. This
+library is provided for cases requiring an append-mostly file to
+record variable length entries. Most developers will find higher
+level libraries like libpmemobj to be more generally useful.
+
+%package -n libpmemlog-debug
+Summary:        Debug variant of the Persistent Memory Resident Log File library
+Requires:       libpmemlog = %{version}-%{release}
+
+%description -n libpmemlog-debug
+The libpmemlog library provides a pmem-resident log file. This
+library is provided for cases requiring an append-mostly file to
+record variable length entries. Most developers will find higher
+level libraries like libpmemobj to be more generally useful.
+
+This sub-package contains debug variant of the library, providing
+run-time assertions and trace points. The typical way to access the
+debug version is to set the environment variable LD_LIBRARY_PATH to
+%{_lib64dir}/pmdk_debug.
```

Comment:
Nit: references to %{_lib64dir} throughout these descriptions should be changed to %{_libdir}